docs/api.rst

```diff
@@ -10279,7 +10272,7 @@ Toggle spam status in evm tokens
 
 .. http:post:: /api/(version)/assets/evm/spam/
 
-   Doing a POST on this endpoint will mark the provided token as a spam token. Any protocol value that the token might have will be overwritten.
+   Doing a POST on this endpoint will mark the provided token as a spam token. It will move the token to the list of ignored assets and from the list of whitelisted tokens too. Any protocol value that the token might have will be overwritten.
```

Comment:
This makes no sense in English. What will this do?

- [X] It will move the token to the list of ignored assets  -- makes sense
- [ ] (will move?) and from the list of whitelisted tokens too -- what?